### PR TITLE
Use YAML template from arcus-azure/azure-devops-templates to push preview package

### PR DESCRIPTION
**Pull request by tomkerkhove**
Use YAML template from arcus-azure/azure-devops-templates to push preview package

### CODE REVIEW

File: build/myget-release.yml [hunk 1]
```diff
@@ -153,10 +160,4 @@ stages:
             inputs:
               artifact: 'Build'
               path: '$(Build.SourcesDirectory)'
-          - task: NuGetCommand@2
-            displayName: 'Push to MyGet.org'
-            inputs:
-              command: push
-              packagesToPush: 'src/**/*.nupkg'
```
Comment (stijnmoreels) [inline build/myget-release.yml]: Would this not be a argument that we should pass in?

Comment (tomkerkhove) [reply build/myget-release.yml]: It's defaulting to that, but you can override if there is a need.

Can add it for sake of being explicit if you want

Comment (stijnmoreels) [reply build/myget-release.yml]: Ah, ok, nevermind then.

**Pull request merged**